src/pathfora.js

```diff
@@ -2256,6 +2255,118 @@
 
   /**
    * @class
+   * @name Inline
+   * @description Inline Personalization
+   */
+  Inline = function () {
+    this.elements = [];
+    this.preppedElements = [];
+
+    /*
+     * @description Prepare all the elements with the attribute provided
+     * @param {attr} name of attribute to select by
+     */
+    this.prepElements = function (attr) {
+      var dataElements = {};
+
+      if (this.elements.length === 0) {
+        this.elements = document.querySelectorAll('[' + attr + ']');
+      }
+
+      var elements = this.elements;
+
+      for (var i = 0; i < elements.length; i++) {
+        if (elements[i].getAttribute(attr) !== null) {
+          var theElement = elements[i],
+              group = theElement.getAttribute('data-liogroup'),
+              type = theElement.getAttribute('data-liotype'),
+              trigger = theElement.getAttribute('data-liotrigger');
+
+          if (!group) {
+            group = 'default';
+          }
+
+          if (!dataElements[group]) {
+            dataElements[group] = [];
+          }
+
+          dataElements[group].push({
+            elem: theElement,
+            displayType: theElement.style.display,
+            group: group,
+            type: type,
+            trigger: trigger
+          });
+        }
+      }
+
+      return dataElements;
+    };
+
+    /*
+     * @description Show/hide the elements based on membership
+     */
+    this.procElements = function () {
+      var elementObj = this.prepElements('data-liotrigger');
```

Comment:
We could do this, but currently the CSS addition is in the constructor for inline should actually get added before `inline.procElements` is ever called.  I actually think this is a good thing though. `inline.procElements` is called only once the dom content has been loaded https://github.com/lytics/pathforajs/blob/migrate-outlio/src/pathfora.js#L2416 and if we tried to look for the existence of `data-liotrigger` elements before adding the css to hide them, the dom content would have to already be loaded, thus I think we'd get that annoying flicker on load, where the element is shown and then quickly hidden by the css addition. :/